deluge/core/core.py

```diff
@@ -724,7 +730,7 @@ def resume_torrent(self, torrent_id: str) -> None:
     @export
     def resume_torrents(self, torrent_ids: List[str] = None) -> None:
         """Resumes a list of torrents"""
-        if not torrent_ids:
+        if torrent_ids is None:
```

Comment:
This changes the functionality, an empty list would return nothing

Reply:
Oh yeah, good point.